R/get_post_samp.R

```diff
@@ -1,9 +1,10 @@
 #adapted from stockassessment::procres
+# This function is not used anymore (right?)
```

Comment:
Yes, this is a function yet to be implemented. It will work with the do_post_samp flags on the cpp side to generate a sample of random effects for diagnostics kind of analogous to the OSA residuals. This was also covered in the Thygesen et al. OSA residual paper.

Reply:
Thanks you both! Yes I figured. I modified it anyway, but it is worth checking if it works as expected in the future.